### DISCUSSION TITLE
2N SIP Mic - API Firmware Update

### DISCUSSION THREAD

**Discussion by igormanoel** [AxisCommunications/discussions#11] [VAPIX] [answered]
Hello,

I was directed to open a ticket here about the possibility of a firmware update to the [2N SIP Mic](https://www.axis.com/products/2n-sip-mic), where it would include a API making possible to change parameters within the microphone with GET HTTP requests.

**Comment by mattias-kindborg-at-work** [comment]
Hi @igormanoel! Thank you for the feedback regarding the 2N SIP Mic API. Let me see if I can get an answer regarding your request.

**Comment by mattias-kindborg-at-work** [comment]
Hi again @igormanoel! What parameters would you benefit from being able to update? Could you elaborate somewhat about the use cases you are trying to solve, it would help us in helping you. Also, if there is anything else you think could be important for us in working with this, please let us know.

**Reply by igormanoel** [reply]
Hello @mattias-kindborg-at-work! 
Ok so this is my problem: I'm involved in a big project which has more than 60 different sites with up to 05 network speakers each, and the client wants to manage all these sites from 03 different control rooms and each of these control rooms have two 2N SIP Mic API. 
The issue is we have more sites than buttons available on the microphone (it has only 12). 
My plan is to develop a software that handles requests from the VMS and translates them to the microphone, changing the speakers that are being called when a specific button is pressed on the microphone. the bare minumum to be able to achieve this is by being able to edit the parameters in "Zones" in the microphone via HTTP API,  and being able to also change the in "Buttons" would help as well.


![image](https://user-images.githubusercontent.com/21324868/175351063-0e456792-babe-48a9-a7e5-35bf80392fdc.png)
![image](https://user-images.githubusercontent.com/21324868/175351849-5f7ecbe3-7d43-45bd-89cb-cc85ba6407c8.png)
![image](https://user-images.githubusercontent.com/21324868/175351909-638680f5-bf12-4aae-9dce-3b05d8c06522.png)
![image](https://user-images.githubusercontent.com/21324868/175351931-243df62d-b36b-4448-98bf-76b67219dd9d.png)

**Comment by mattias-kindborg-at-work** [comment]
Hi @igormanoel! We're currently investigating this, however I cannot give you a timeline. Just wanted to give you a heads up that this discussion will not have an answer for a couple of more weeks.

**Reply by igormanoel** [reply]
Thank you for the response @mattias-kindborg-at-work !

I'll be looking foward to your conclusions.

Thanks in advance.

**Comment by igormanoel** [comment]
Hello @mattias-kindborg-at-work is there any news about this matter?

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @igormanoel.

Got some feedback from the team. They recognize that this is a problem other customers are experiencing as well, but unfortunately they don't have any plans to fix the problem in this specific product. What other customers usually do is to work around the issue by using more than one SIP mic. I'm sorry.
